kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wdt/create-domain-script.sh

```diff
@@ -0,0 +1,217 @@
+#   given Oracle home, WDT model file, WDT vars file, etc.
+#
+# Usage:
+#
+#   Export customized values for the input shell environment variables as needed
+#   before calling this script.   See also README.
+#
+# Outputs:
+#
+#   WDT install:           WDT_DIR/weblogic-deploy/...
+#
+#   Copy of wdt model:     WDT_DIR/$(basename WDT_MODEL_FILE)
+#   Copy of wdt vars:      WDT_DIR/$(basename WDT_VAR_FILE)
+#
+#   WDT logs:              WDT_DIR/weblogic-deploy/logs/...
+#   WDT stdout:            WDT_DIR/createDomain.sh.out
+#
+#   WebLogic domain home:  DOMAIN_DIR/<domain name from model>/...
+#
+# Input environment variables:
+#
+#   ORACLE_HOME    Oracle home with a WebLogic install.
+#                  default:  /u01/oracle
+#
+#   WDT_MODEL_FILE Full path to WDT model file.
+#                  default:  the directory that contains this script
+#                            plus "/wdt_model.yaml"
+#
+#   WDT_VAR_FILE   Full path to WDT variable file (java properties format).
+#                  default:  the directory that contains this script
+#                            plus "/create-domain-inputs.yaml"
+#
+#   WDT_INSTALL_ZIP_FILE  Filename of WDT install zip.
+#                  default:  weblogic-deploy.zip
+#
+#   WDT_INSTALL_ZIP_URL   URL for downloading WDT install zip
+#                  default:  https://github.com/oracle/weblogic-deploy-tooling/releases/download/weblogic-deploy-tooling-0.9/$WDT_INSTALL_ZIP_FILE
+#
+#   https_proxy    Proxy for downloading WDT_INSTALL_ZIP_URL.
+#                  default: "http://www-proxy-hqdc.us.oracle.com:80"
+#                  (If set to empty the script will try with no proxy.)
+#
+#   https_proxy2   Alternate proxy for downloading WDT_INSTALL_ZIP_URL
+#                  default: ""
+#                  (If set to empty the script will try with no proxy.)
+#
+#   WDT_DIR        Target location to install and run WDT, and to keep a copy of
+#                  $WDT_MODEL_FILE and $WDT_MODEL_VARS. Also the location
+#                  of WDT log files.
+#                  default:  /shared/wdt
+#
+#   DOMAIN_DIR     Target location for generated domain. When wdt completes,
+#                  this will contain a directory named after the domain
+#                  defined in the WDT_MODEL_FILE.
+#                  default:  /shared/domain
+#
+
+# Initialize globals
+
+export ORACLE_HOME=${ORACLE_HOME:-/u01/oracle}
+
+SCRIPTPATH="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
+WDT_MODEL_FILE=${WDT_MODEL_FILE:-"$SCRIPTPATH/wdt_model.yaml"}
+WDT_VAR_FILE=${WDT_VAR_FILE:-"$SCRIPTPATH/create-domain-inputs.yaml"}
+
+WDT_DIR=${WDT_DIR:-/shared/wdt}
+
+DOMAIN_DIR=${DOMAIN_DIR:-/shared/domain}
+
+WDT_INSTALL_ZIP_FILE="${WDT_INSTALL_ZIP_FILE:-weblogic-deploy.zip}"
+WDT_INSTALL_ZIP_URL=${WDT_INSTALL_ZIP_URL:-"https://github.com/oracle/weblogic-deploy-tooling/releases/download/weblogic-deploy-tooling-0.9/$WDT_INSTALL_ZIP_FILE"}
+
```

Comment:
We should probably update the default wdt install zip version to the latest version and test it.   I think I was using 0.9 as that was the latest at the time.  I think the latest version is up to 11.

Reply:
Derek posts new version numbers in wdt-deploy-tooling slack when he puts them out.

Reply:
Good point. The latest version is 0.12. I'll update.

Reply:
Update is done.